Apromore-Boot/src/main/resources/menu.json

```diff
@@ -0,0 +1,93 @@
+{
+  "groups": [
+    {
+      "id": "FILE",
+      "items": [
+        {
+          "id": "UPLOAD"
+        },
```

Comment:
Menu is part of the Portal (it is the Portal menu). Putting it in another module like Apromore-Boot instead of the Portal module adds to the complexity of the system, a phenomenon called "Unknown Unknowns": i.e. you don't know where the things (you don't know) are, how it works, how to modify when needed, etc. 
Suggest put this menu.json file in the Portal under the same package of all menu related code.

Reply:
Thanks, Bruce. Initially, I thought that the configuration should be configurable in the runtime and hence I put it in the ApromoreBoot. But I agree with your point that we probably should have default in the portal as well.